modules/tinymce/src/core/test/ts/browser/EditorSizeTest.ts

```diff
@@ -0,0 +1,76 @@
+import { context, describe, it } from '@ephox/bedrock-client';
+import { Height } from '@ephox/sugar';
+import { TinyDom, TinyHooks } from '@ephox/wrap-mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.EditorSizeTest', () => {
+  context('Minimum Size Enforcement', () => {
+    context('Unitless wrapper', () => {
+      const hook = TinyHooks.bddSetup<Editor>({
+        selector: 'textarea',
+        height: 10,
+        min_height: 500,
+        base_url: '/project/tinymce/js/tinymce'
+      }, []);
+      it('TINY-11108: unitless', () => {
```

Comment:
I think it's easier to read if we separate the setup of the hook from the `it` call.

```suggestion
      }, []);

      it('TINY-11108: unitless', () => {
```